Bruin Bite/View Controllers/PreferenceViewController.swift

```diff
@@ -29,8 +29,8 @@ class PreferenceViewController: UIViewController, MatchDelegate {
     var generatedMatchID: Int? = nil // note:  if set, means that we can segue to searching screen. If not set, then we have a problem.
     
     @IBAction func MatchButton(_ sender: Any) {
-        
-        if(chosen.isEmpty || meal_day == "" || dining_halls.isEmpty || MAIN_USER.accessUserInfo(type: "period") == "") {
+        //TODO: Must check that the User has chosen a meal period!
```

Comment:
Make this comment more explanatory - what does this mean?  Maybe leave your old code commented out so we know what was supposed to happen in addition to clarifying exactly what needs to happen here.